### PR TITLE
Hotfix with a sanity test for api-gateway

### PR DESCRIPTION
**Pull request by akuleshov7**
### What's done:
- fixed gateway typo
- due to the change in snakeyaml API (2.0), api-gateway is not able to start: `Caused by: java.lang.NoSuchMethodError: org.yaml.snakeyaml.constructor.SafeConstructor: method 'void <init>()' not found`

### CODE REVIEW

File: api-gateway/build.gradle.kts [hunk 1]
```diff
@@ -12,4 +12,13 @@ dependencies {
     implementation(libs.spring.cloud.starter.kubernetes.client.config)
     implementation(libs.spring.security.core)
     implementation(projects.authenticationService)
+
+    // FixMe: we officially bring serialization CVE to our project
```
Comment (akuleshov7) [inline api-gateway/build.gradle.kts:16]: Say hello to a back door!

Comment (kgevorkyan) [reply api-gateway/build.gradle.kts:16]: does the test failed on CI, without this?

File: api-gateway/src/test/kotlin/com/saveourtool/save/gateway/StartupTest.kt [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import org.junit.jupiter.api.Test
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.test.context.junit.jupiter.SpringJUnitConfig
+
+
+@SpringBootTest
+@SpringJUnitConfig
+class StartupTest {
+    @Test
+    fun contextLoads() {
```
Comment (kgevorkyan) [inline api-gateway/src/test/kotlin/com/saveourtool/save/gateway/StartupTest.kt:12]: @0x6675636b796f75676974687562 would say here `fun contextLoads() = Unit` 😄

Comment (akuleshov7) [reply api-gateway/src/test/kotlin/com/saveourtool/save/gateway/StartupTest.kt:12]: btw, great suggestion

**Review by unix-junkie — APPROVED** [review]

**Review by 0x6675636b796f75676974687562 — APPROVED** [review]

**Pull request merged**